template-manager-cmtemplate/src/main/java/com/sequenceiq/cloudbreak/cmtemplate/configproviders/yarn/YarnCloudStorageRoleConfigProvider.java

```diff
@@ -15,19 +15,17 @@
 @Component
 public class YarnCloudStorageRoleConfigProvider extends AbstractRoleConfigProvider {
 
-    private static final String NODEMANAGER_CONFIG_SAFETY_VALVE = "nodemanager_config_safety_valve";
+    private static final String NODEMANAGER_REMOTE_APP_LOG_DIR_TEMPLATE_PARAM = "yarn_nodemanager_remote_app_log_dir";
 
     private static final String NODEMANAGER_REMOTE_APP_LOG_DIR = "yarn.nodemanager.remote-app-log-dir";
 
     @Override
     protected List<ApiClusterTemplateConfig> getRoleConfigs(String roleType, TemplatePreparationObject source) {
         switch (roleType) {
             case YarnRoles.NODEMANAGER:
-                String cloudStorageProperty = getCloudStorageProperty(source);
```

Comment:
Oh, actually I think the overridden `getCloudStorageProperty()` below is not needed anymore.

Reply:
Yes. removed the same.